### DISCUSSION TITLE
if (FreshRSS_Context::$system_conf->allow_robots)

### DISCUSSION THREAD

**Discussion by robotfx** [FreshRSS/FreshRSS#4350] [Q&A] [answered]
In the layout.phtml there is the line 
`if (FreshRSS_Context::$system_conf->allow_robots)
`Where can I allow/disallow this in the settings? I cannot find any option, so for now I've modified the theme file, but I assume I missed some check box in the administrating area :)

**Comment by Alkarex** [comment, marked as answer]
Such rare and/or advanced options can be set in your `./data/config.php` file.
See model:

https://github.com/FreshRSS/FreshRSS/blob/ae097f5bd82aa24ea19cca72ac9d1594b0dcaec0/config.default.php#L83-L84
